lib/project_types/script/script_project.rb

```diff
@@ -2,14 +2,15 @@
 
 module Script
   class ScriptProject < ShopifyCli::Project
-    attr_reader :extension_point_type, :script_name, :language, :description
+    attr_reader :extension_point_type, :script_name, :language, :description, :configuration_ui_yaml
```

Comment:
Small nit here on naming.

I know that in the backend we have already named this `configurationUiYaml` but what do you think of instead of `configurationUiYaml` we just default to `configurationUi`? I'm suggesting this mostly because:

- I don't think adding `Yaml` adds any relevant meaning to the property name
- Excluding `Yaml` will allow us to potentially swap the internal config representation without having to rename the property name (in the backend if we every swap Yaml for JSON, it will require a migration for example)

I'm advocating for this change in the CLI only (just to keep it scoped to this change)

Reply:
Yeah thats a good idea, thanks! I'll update that

Reply:
I figured it makes sense to do that now for both the queries and the CLI before anything starts being consumed, so I opened a server side PR too and updated everything here